README.md

```diff
@@ -19,22 +19,31 @@ You can either use Donut directly or check out the available plugins:
 ## Release Notes
 See what's new [here](release-notes.md)
 
-### download
+### Download
 ```
 wget http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar
 ```
 or download the latest release from: [here](http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar)
 
-### run from command line
+### Run from command line
 
 ```
 java -jar donut-<Version>.jar -s cucumber:/my/path/cucumber-reports -n myProjectName
 ```
 
-### options
+### Run using Docker
+
+Donut is packaged as a [Docker container](https://hub.docker.com/r/donutreport/donut-report/)
+
+To run using Docker:
+```
+docker run -v /path/to/your/cucumber-reports:/source -v /path/to/output-report:/output donutreport/donut-docker -n myProjectName [options]

```

Comment:
The input and output volumes are required to ensure the folders are available to the container, and aren't lost when the report is generated.

Reply:
I've updated so you need to specify the `-s` command yourself. 
Updated the README